modules/SofaGeneralAnimationLoop/MechanicalMatrixMapper.inl

```diff
@@ -336,7 +338,8 @@ void MechanicalMatrixMapper<DataTypes1, DataTypes2>::addKToMatrix(const Mechanic
 
     sofa::simulation::Node *node = l_nodeToParse.get();
     size_t currentNbInteractionFFs = node->interactionForceField.size();
-
+    msg_info() << "nb m_nbInteractionForceFields :" << m_nbInteractionForceFields;
+    msg_info() << "nb currentNbInteractionFFs :" << currentNbInteractionFFs;
```

Comment:
It is better to:
    msg_info() << "nb m_nbInteractionForceFields :" << m_nbInteractionForceFields << msgendl
                        << "nb currentNbInteractionFFs :" << currentNbInteractionFFs;

Reply:
Thank you